bigquery/google/cloud/bigquery/table.py

```diff
@@ -808,18 +824,38 @@ def items(self):
         return items
 
     def get(self, key, default=None):
-        """
-        Return value under specified key
-        Defaults to None or specified default
-        if key does not exist:
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('x')
-        'a'
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('z')
-        None
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('z', '')
-        ''
-        >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('z', default = '')
-        ''
+        """Return value for ``key`` if ``key`` is in the row, else
+        ``default``.
+
+        Args:
+            key (str): The key of the column to access
+
+            default (object):
+                The default value to use if the key does not exist. (Defaults
+                to ``None``.)
+
+        Returns:
+            object:
+                The value associated with the provided key, or a default value.
+
+        Examples:
+            When the key exists, the value associated with it is returned.
+
+            >>> Row(('a', 'b'), {'x': 0, 'y': 1}).get('x')
+            'a'
+
+            The default value is ``None`` when the key does not exist.
```

Comment:
For your future reference

```
:data:`None`
```

will actually link to the std. library, rather than just code font. But I doubt people need that link.

Reply:
Good to know, thanks